MobiFlight/Board.cs

```diff
@@ -279,6 +328,37 @@ public class Board : IMigrateable
         /// </summary>
         public UsbDriveSettings UsbDriveSettings;
 
+        /// <summary>
+        /// Base path for custom firmware
+        /// </summary>
+        public String BasePath;
+
+        /// <summary>
+        /// The image resource
+        /// </summary>
+        public Image BoardImage;
+
+        /// <summary>
+        /// Returns the correct Partner Level
+        /// </summary>
+        /// <returns></returns>
+        [JsonIgnore]
+        public BoardPartnerLevel PartnerLevel { get {
```

Comment:
You could add unit tests for this property as well, to check that it gets set to one of the three enum values for every file loaded.

Reply:
Good idea!